graf2d/gpadv7/inc/ROOT/RAttrText.hxx

```diff
@@ -74,8 +74,8 @@ class RAttrText : public RAttrBase {
       }
 
       SetFontFamily(family);
-      SetFontWeight(style);
-      SetFontStyle(weight);
+      SetFontWeight(weight);
+      SetFontStyle(style);
```

Comment:
OK we need strongly typed somethings here :-)

Reply:
Yes, for all kinds of line/fill/marker styles one have to provide `enum` or special typedefs. 
Just a peace of work, for which I never find time.